napps/kytos/of_stats/stats.py

```diff
@@ -5,13 +5,14 @@
 
 import rrdtool
 from kytos.core import KytosEvent, log
-from kytos.core.flow import Flow
+from napps.kytos.of_core.flow import Flow
```

Comment:
@renanrodrigo I think that pylint would fail over here, this import should be done after importing from 'external' libraries.